File: TAREA 03/Ejercicio1.cpp
#include <iostream>
using namespace std;

int main() {
    int N;
    cout << "Ingrese el valor de N: ";
    cin >> N;

    // Inicialización de variables de control
    int contador = 1;   // Controla el ciclo e indica el número natural actual
    int acumulador = 0; // Guarda la suma total de los números

    // Bucle iterativo con la sentencia while
    while (contador <= N) {
        acumulador += contador; // Se suma el valor actual del contador al acumulador
        contador++;             // Se incrementa el contador para pasar al siguiente número
    }

    cout << "La sumatoria de los primeros " << N << " numeros naturales es: " << acumulador << endl;

    return 0;
}

File: TAREA 03/Ejercicio1.py
# Solicitamos el valor de N al usuario
N = int(input("Ingrese el valor de N: "))

# Inicialización de variables de control
contador = 1    # Controla el ciclo e indica el número natural actual
acumulador = 0  # Guarda la suma total de los números

# Bucle iterativo con la sentencia while
while contador <= N:
    acumulador += contador  # Se suma el valor actual del contador al acumulador
    contador += 1           # Se incrementa el contador para pasar al siguiente número

print(f"La sumatoria de los primeros {N} números naturales es: {acumulador}")